packages/memory/integration-tests/src/generic-memory-worker.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
```

Comment:
Does this need to be in version control?

Reply:
removed it